### ISSUE TITLE
Error: Batch normalization gradient requires mean and offset to have equal ranks.

### ISSUE THREAD

**Issue by GU8187** [tensorflow/tfjs#2120]
Hello Everyone,

[`pix2pix`][`pix2pix.js`][`tensorflow.js`]
I'm trying to set up online environment for my new learned pix2pix model. I have exported the .pict model with the guides kindly provided by @yining1023 & Dongphil Yoo.

#### TensorFlow.js version
https://cdn.jsdelivr.net/npm/@tensorflow/tfjs

#### Browser version
Google Chrome Version 77.0.3865.90 (Build officiel) (64 bits)
FIrefox quantum 69..0.1

#### Describe the problem or feature request
After .pict Model is loaded
Anything done with tools (Eraser/clear/pencil) produces error as following in Console

tfjs:2 Uncaught (in promise) Error: Batch normalization gradient requires mean and offset to have equal ranks.
    at f (tfjs:2)
    at As (tfjs:2)
    at Object.batchNorm (tfjs:2)
    at batchnorm (pix2pix.js:34)
    at pix2pix.js:64
    at tfjs:2
    at t.scopedRun (tfjs:2)
    at t.tidy (tfjs:2)
    at Object._e [as tidy] (tfjs:2)
    at Pix2pix.transfer (pix2pix.js:47)

#### Code to reproduce the bug / link to feature request
http://guillaumedamry.com/p2pampa/

Any hint on this (programming or orienting request on github) would be much appreciated because I am completly stuck.

I have compiled with python3 [`pix2pix`][`pix2pix.js`][`tensorflow.js`]

**Comment by caisq** [comment]
FYI, https://github.com/tensorflow/tfjs/pull/2114 contains a workaround to this issue for the implementation of `LayerNormalization`

**Comment by GU8187** [comment]
Thanks for your answer still I'm a bit confused...

- Should there be an update to [`pix2pix_tensorflowjs`] repo 

- or should I update locally in my pix2pix-tensorflow/.git (before compiling a new .pict model), 

- or is this is a next master of tensorflowjs, so I should replace <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script> with a proper path, with local hosting of library's brand new version ?

Regards

**Comment by caisq** [comment]
Just to be clear #2114 is not meant to be a fix to this problem. It's just relevant to the problem and shows how a user may *in principle* workaround it. The fix will come in a separate PR.

**Comment by GU8187** [comment]
Got it thanks, look forward to it !

**Comment by GU8187** [comment]
@dsmilkov Daniel, can you give me a hint on how to set the environment straight : what should I update, or change in my settings ?

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
